forge/routes/api/shared/users.js

```diff
@@ -41,21 +47,41 @@ module.exports = {
             } else if (request.body.name === '') {
                 user.name = request.body.username || user.username
             }
-            if (request.body.email) {
-                if (user.sso_enabled && user.email !== request.body.email) {
+            if (request.body.email && user.email !== request.body.email) {
+                // SSO cannot change email address
+                // TODO: consider also making make field non-editable in frontend if sso_enabled
```

Comment:
It already is non-editable if sso_enabled - at least, in the user's own account settings view.

https://github.com/flowforge/flowforge/blob/4e8e2b63f3092efe03763b62a882984dba40708e/frontend/src/pages/account/Settings.vue#L208